docs/getting_started/kernel.md

```diff
@@ -0,0 +1,337 @@
+		WithConfigFileFlag,
+		WithConfigEnvKeyReplacer(cfg.DefaultEnvKeyReplacer),
+		WithConfigSanitizers(cfg.TimeSanitizer),
+		WithMetadataServer,
+		WithConsumerMessagesPerRunnerMetrics,
+		WithKernelSettingsFromConfig,
+		WithLoggerApplicationTag,
+		WithLoggerContextFieldsMessageEncoder,
+		WithLoggerContextFieldsResolver(log.ContextLoggerFieldsResolver),
+		WithLoggerHandlersFromConfig,
+		WithLoggerMetricHandler,
+		WithLoggerSentryHandler(log.SentryContextConfigProvider, log.SentryContextEcsMetadataProvider),
+		WithMetricDaemon,
+		WithProducerDaemon,
+		WithTracing,
+		WithUTCClock(true),
+	}
+
+	options = append(defaults, options...)
+
+	return New(options...)
+}
+```
+
+Notice the config _Options_ for the default _Kernel_, explained below:
+
+| Option | Description |
+| ------ | ----------- |
+| WithApiHealthCheck | Adds the "api-health-check" module to this _Kernel_ |
+| WithConfigErrorHandlers | Configures a custom _ErrorHandler_ |
+| WithConfigFile | Reads additional configurations from a given file |
+| WithConfigFileFlag | If the app was started with a config flag, reads additional configurations from the location stored in that flag |
+| WithConfigEnvKeyReplacer | Allows a replacer to first process config values |
+| WithConfigSanitizers | Adds configuration sanitizers, in this case the cfg.TimeSanitizer formats to time.RFC3339 |
+| WithMetadataServer | Adds the "metadata-server" module, exposing endpoints like "/config" |
+| WithConsumerMessagesPerRunnerMetrics | Adds the "stream-metric-messages-per-runner" module, which repeatedly writes a metric regarding how many messages were processed |
+| WithKernelSettingsFromConfig | Adds the KillTimeout option taken from settings.KillTimeout |
+| WithLoggerApplicationTag | Adds the logger "application" field, taken from config.GetString("app_name") |
+| WithLoggerContextFieldsMessageEncoder | Adds a default logger encoder |
+| WithLoggerContextFieldsResolver | Adds a default logger context fields resolver|
+| WithLoggerHandlersFromConfig | Adds an extra log handler from config |
+| WithLoggerMetricHandler | Adds a default log metric handler |
+| WithLoggerSentryHandler | Adds a default logger sentry handler |
+| WithMetricDaemon | Adds the "metric" module to this _Kernel_ |
+| WithProducerDaemon | Adds a default producer daemon |
+| WithTracing | Adds a default logger tracingHandler |
+| WithUTCClock | Makes the _Kernel_ use clock.WithUseUTC(useUTC) |
+
+Similarly, package _application_ offers two helper functions for running a module, that internally will make used of `application.Default()`:
+
+[embedmd]:# (../../pkg/application/runners.go /func RunModule/ /\n}/)
+```go
+func RunModule(name string, moduleFactory kernel.ModuleFactory, options ...Option) {
+	app := Default(options...)
+	app.Add(name, moduleFactory)
+	app.Run()
+}
+```
+
+This functions instantiates the `Default()` kernel, adds all options to it, adds the module then runs that kernel.
```

Comment:
I think it would be more useful to explain the reasons for methods than repeat what they do in text after just a few lines of code (which are not really hard to understand).